tests/test_parser.py

```diff
@@ -1,53 +1,138 @@
+# coding: utf-8
+
 from __future__ import unicode_literals
 import unittest
 
 import twingly_search
 
 class ParserTest(unittest.TestCase):
+    def get_fixture(self, fixture_name):
+        file_path = "./tests/fixtures/%s.xml" % fixture_name
+        fixture = open(file_path, 'r').read()
+        return fixture
+
+    def assert_blog_posts_equal(self, actual_post, expected_post):
```

Comment:
> How about implementing __eq__ on post instead.

TIL about `__dict__` :)

One disadvantage is that it becomes impossible to find out whats wrong by looking at the test output:

```shell
AssertionError: 'Skvallernytt.se' != u'FEL'

# becomes:

AssertionError: <twingly_search.post.Post object at 0x10a8b76d0> != <twingly_search.post.Post object at 0x10a8b75d0>
```